apps/emqx_bridge/src/emqx_bridge_resource.erl

```diff
@@ -71,6 +71,8 @@ bridge_id(BridgeType, BridgeName) ->
     Type = bin(BridgeType),
     <<Type/binary, ":", Name/binary>>.
 
+parse_bridge_id(<<"bridge:", BridgeId/binary>>) ->
+    parse_bridge_id(BridgeId);
```

Comment:
bridge:xxx:xxx is a resource id, not a bridge-id.

Reply:
Ok, thanks. Will fix!

Reply:
I think @zmstone already fixed in the commit below